Workbooks/Azure Stack HCI/AtScale/Insights.workbook

```diff
@@ -17,7 +17,7 @@
             "type": 4,
             "isRequired": true,
             "value": {
-              "durationMs": 14400000
+              "durationMs": 86400000
```

Comment:
Is this change needed?

Reply:
This is just the time picker. This won't affect anything. Made it to last 24hrs instead of last 4hrs.